test/docs/OpenJ9TestUserGuide.md

```diff
@@ -109,6 +109,7 @@ Please read [DependentLibs.md](./DependentLibs.md) for details.
       - level:   [sanity|extended] (extended default value)
       - group:   [functional|system|openjdk|external|perf|jck] (required 
                  to provide one group per test)
+      - type:    [regular|native] (regular default value)
```

Comment:
I think we need to explain what does `native` mean.
For example, if a test is tagged with native, it means this test needs to run with test image (native test libs). `NATIVE_TEST_LIBS` needs to be set for local testing. If Grinder is used, native test lib download link needs to be provided in addition to SDK download link in `CUSTOMIZED_SDK_URL`. https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins